### ISSUE TITLE
From NYR 2025: demonstrating overfitting

### ISSUE THREAD

**Issue by topepo** [tidymodels/workshops#397]
In the notes, we have them fit a classification tree, predict the training and test set, and compute the Brier score. The results: 

Brier: 

 - train: 0.113
 - test: 0.208
 - resampled: 0.214

That does show some differences, but the problem is that 0.113 is a pretty mediocre Brier score value. It doesn't really show overfitting. 

We should consider showing the overfitting via the ROC AUC: 

ROC: 

 - train: 0.881
 - test: 0.703
 - resampled: 0.704

That seems like a stronger argument.
